components/productImage/productImage.js

```diff
@@ -0,0 +1,62 @@
+import  React ,{ Component } from "react";
```

Comment:
Components Always start with Capital letter . Change the filenames accordingly and ProductImage is not a good name